<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PurchaseOrders</class>
 <widget class="QMainWindow" name="PurchaseOrders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LPOs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/images/erp-pics/Move.png</normaloff>:/images/erp-pics/Move.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="CustomQueryView" name="trvCustomers"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="cmdNewLPO">
            <property name="text">
             <string>New LPO</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="horiSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="CustomQueryView" name="trvOrders">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="cmdUploadLPOFile">
             <property name="text">
              <string>Upload LPO File</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cmdDownloadLPO">
             <property name="text">
              <string>Download LPO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cmdEmailLPO">
             <property name="text">
              <string>Email LPO</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cmdAssignOCs">
             <property name="text">
              <string>Assign OCs</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="vertiSplitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Items In This Order</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CustomQueryView" name="trvOrderDetails">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="cmdNewJobTicket">
              <property name="text">
               <string>New Job Ticket For Selected Product</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Job Tickets in this Order:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CustomQueryView" name="trvOcs"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>trvCustomers</tabstop>
  <tabstop>cmdNewLPO</tabstop>
  <tabstop>trvOrders</tabstop>
  <tabstop>cmdUploadLPOFile</tabstop>
  <tabstop>cmdDownloadLPO</tabstop>
  <tabstop>cmdEmailLPO</tabstop>
  <tabstop>cmdAssignOCs</tabstop>
  <tabstop>trvOrderDetails</tabstop>
  <tabstop>trvOcs</tabstop>
  <tabstop>cmdNewJobTicket</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
